FFCDemoPaymentService/Scheduling/SchedulerService.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.Collections.Generic;
+using FFCDemoPaymentService.Models;
+using FFCDemoPaymentService.Data;
+
+namespace FFCDemoPaymentService.Scheduling
+{
+    public class ScheduleService : IScheduleService
+    {
+        private readonly ApplicationDbContext db;
+        public ScheduleService(ApplicationDbContext db)
+        {
+            this.db = db;
+        }
+
+        public void CreateSchedule(string claimId)
+        {
+            DateTime startDate = DateTime.Now.Date;
+
+            try
+            {
+                db.Schedule.AddRange(ScheduleBuilder(claimId, startDate));
+                db.SaveChanges();
+            }
+            catch(Exception ex)
+            {
+                throw new Exception(ex.Message, ex.InnerException);
+            }
+        }
+
+        private List<Schedule> ScheduleBuilder(string claimId, DateTime startDate)
+        {
+            if(claimId != null && startDate != null)
```

Comment:
No need for this `if`.  Both parameters are mandatory so will never be null or the code won't even compile.